docs/src/pages/examples/rag/filter-rag.mdx

```diff
@@ -128,7 +138,11 @@ export const ragAgent = new Agent({
 });
 ```
 
-We can provide an example of how the metadata is structured for the agent to use when filtering with the tool.
+The agent's instructions are designed to:
```

Comment:
I'm confused, at what point do we actually give the agent the tool?

Reply:
You should also say: "EVEN THOUGH we are giving the agent the tools we ALSO need to give them this detailed system prompt." (w/o the caps obviously)